release-notes/release-notes.json

```diff
@@ -1,4 +1,34 @@
 [
+    {
+        "version": "3.0.1",
+        "title": "Fiddler Everywhere v3.0.1",
+        "date": "December 03, 2021",
+        "items": [
+            {
+                "type": "Fixed",
+                "entries": [
+                    {
+                        "title": "The app freezes after inspecting request with form-data body"
+                    },
+                    {
+                        "title": "Screen for inability to merge accounts has incorrect content"
+                    },
+                    {
+                        "title": "Unable to use the application when quota exceeded dialog is shown"
+                    },
+                    {
+                        "title": "The app freezes when there's a filter on 'Rules modified' column and app is restarted"
+                    },
+                    {
+                        "title": "Shared Rulesets might not be editable by the owner"
+                    },
+                    {
+                        "title": "Tab selection and grid are missing when the session does not have data for the currently selected inspector"
```

Comment:
Tab selection and grid within the Inspectors are missing when the session does not have data for the currently selected inspector